Sources/RoutingKit/TrieRouter.swift

```diff
@@ -53,7 +66,7 @@ public final class TrieRouter<Output>: Router, CustomStringConvertible {
 
         // if this node already has output, we are overriding a route
         if current.output != nil {
-            print("[Routing] Warning: Overriding route output at: \(path.string)")
+            self.logger.warning("[Routing] Overriding route output at: \(path.string)")
```

Comment:
No need for `[Routing]`.
Vapor does:
```
self.configuration.logger.notice("Server starting on \(addressDescription)")
```
Which shows up as 
```
2023-08-15T12:09:01+0330 notice codes.vapor.application : [Vapor] Server starting on http://127.0.0.1:8080
```
Which means it's the console-logger which is adding the `[Vapor]`.

Reply:
Should come from console kit right here:
```
if self.logLevel <= .trace {
    text += "[ \(self.label) ] ".consoleText()
}
```

Reply:
Which means the text only appears when logging is set to `trace` level - `trace` is level `1`, numerically, there aren't any lower-numbered levels.

Reply:
So i'm wrong but because the thing is from the stdout log handler i think, not from the console logger.